samples/tutorials/mnist/README.md

```diff
@@ -0,0 +1,110 @@
+how they combine in the form of a graph. The pipeline includes the definition of the inputs (parameters) required to 
+run the pipeline and the inputs and outputs of each component. A pipeline component is a self-contained set of user 
+code, packaged as a Docker image, that performs one step in the pipeline. For example, a component can be responsible 
+for data preprocessing, data transformation, model training, and so on. 
+
+## Content Overview:
+In this tutorial, we designed a series of notebooks to demonstrate how to interact with `Kubeflow Pipelines` through
+[Python SDK](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp). In particular
+- [00 Kubeflow Cluster Setup](00_Kubeflow_Cluster_Setup.ipynb): this notebook helps you deploy a Kubeflow 
+cluster through CLI. Note that it is also possible to deploy the Kubeflow cluster though 
+[UI](https://www.kubeflow.org/docs/gke/deploy/deploy-ui/)
+
+Then, notebooks 01-04 use one concrete use case, i.e., 
+[MNIST classification](https://www.tensorflow.org/tutorials/quickstart/beginner), to demonstrate different ways of
+authoring a pipeline component: 
+- [01 Lightweight Python Components](01_Lightweight_Python_Components.ipynb): this notebook demonstrates how to build a 
+component through defining a stand-alone python function and then calling `kfp.components.func_to_container_op(func)` to 
+convert, which can be used in a pipeline.
+
+- [02 Local Development with Docker Image Components](02_Local_Development_with_Docker_Image_Components.ipynb): this 
+notebook guides you on creating a pipeline component with `kfp.components.ContainerOp` from an existing Docker image 
+which should contain the program to perform the task required in a particular step of your ML workflow.
+
+- [03 Reusable Components](03_Reusable_Components.ipynb): this notebook describes the manual way of writing a full 
+component program (in any language) and a component definition for it. Below is a summary of the steps involved in 
+creating and using a component.
+    - Write the program that contains your component’s logic. The program must use files and command-line arguments 
+    to pass data to and from the component.
+    - Containerize the program.
+    - Write a component specification in YAML format that describes the component for the Kubeflow Pipelines system.
+    - Use the Kubeflow Pipelines SDK to load your component, use it in a pipeline and run that pipeline.
+
+- [04 Reusable and Pre-build Components as Pipeline](04_Reusable_and_Pre-build_Components_as_Pipeline.ipynb): this 
+notebook combines our built components, together with a pre-build GCP AI Platform components 
+and a lightweight component to compose a pipeline with three steps.
+    - Train a MNIST model and export it to GCS
+    - Deploy the exported Tensorflow model on AI Platform prediction service
+    - Test the deployment by calling the end point with test data
+
+## Setups Overview:
```

Comment:
modified :)

Reply:
Hey @joeliedtke , I think your suggestion makes sense. But the intention to include this section is to let the reader have a high level feeling what are expected for the setup. How about let me remove all the detail and and keep the list of steps only as follows, and the detail is left to the notebook. HDYT?

```markdown
## Setups Overview:
### Prerequisites
Before you follow the instructions below to deploy your own Kubeflow cluster, you should
- have a [GCP project setup](https://www.kubeflow.org/docs/gke/deploy/project-setup/) for your Kubeflow deployment 
with you having the [owner role](https://cloud.google.com/iam/docs/understanding-roles#primitive_role_definitions) 
for the project and with the following APIs enabled:
    - [Compute Engine API](https://pantheon.corp.google.com/apis/library/compute.googleapis.com)
    - [Kubernetes Engine API](https://pantheon.corp.google.com/apis/library/container.googleapis.com)
    - [Identity and Access Management(IAM) API](https://pantheon.corp.google.com/apis/library/iam.googleapis.com)
    - [Deployment Manager API](https://pantheon.corp.google.com/apis/library/deploymentmanager.googleapis.com)
    - [Cloud Resource Manager API](https://pantheon.corp.google.com/apis/library/cloudresourcemanager.googleapis.com)
    - [AI Platform Training & Prediction API](https://pantheon.corp.google.com/apis/library/ml.googleapis.com)
- have set up [OAuth for Cloud IAP](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/)
- have installed and setup [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
- have installed [gcloud-sdk](https://cloud.google.com/sdk/)

### Setup Environment and Deploy Kubeflow Cluster
* Download [kfctl](https://github.com/kubeflow/kubeflow/releases)
* Create user credentials
* Setup environment variables
* Create dedicated service account for deployment
* Deploy Kubeflow
* Install the lastest version of [kfp](https://kubeflow-pipelines.readthedocs.io/en/latest/)
```

Reply:
This also align with the changes made in Notebook-00

Reply:
I would still recommend removing the Setups Overview section. Users may be confused by the links in this content and try to setup their environment using this section as their instructions. Since your guide provides better instructions in notebook 00, it is better to leave this content until that point. Or, to put the complete instructions in this page and remove notebook 00.